src/polyfills.ts

```diff
@@ -71,3 +71,8 @@ import 'hammerjs/hammer';
 // Needed on browsers with native `customElements`.
 // (E.g.: Chrome, Opera)
 import '@webcomponents/custom-elements/src/native-shim';
+
+// Add global to window, assigning the value of window itself.
+// See https://github.com/angular/angular-cli/issues/9827#issuecomment-386154063
+// Needed for the twemoji library.
```

Comment:
Ah, cool, so this is important I think, and will probably have to be done by every project that includes this component. Want to add some info to the README.md on that? I think I found this and filed some issues related to it when I did the remake of the IO demo. Maybe for a followup PR